staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -275,7 +275,6 @@ func (p *testPatcher) Update(ctx context.Context, name string, objInfo rest.Upda
 }
 
 func (p *testPatcher) Get(ctx context.Context, name string, options *metav1.GetOptions) (runtime.Object, error) {
-	p.t.Fatal("Unexpected call to testPatcher.Get")
```

Comment:
Just for the code here, I think it's fine to leave it as it is. It seems that this `Get` function is just to implement the interface and is not called in any test.

This pr came up because I executed goland's [unreachable code test](https://www.jetbrains.com/help/go/code-inspections-in-go.html#Code_style_issues) on the wrong codebase (I wanted to test our business code, but had the kubernetes code open😳) ....

So I guess if goland didn't think that a Fatal elsewhere would affect the code execution. Then we can leave it as it is for now.

Reply:
I think I'd leave this as is

Reply:
revert this file.

Reply:
I noticed in the go source code that this case would use `panic("unreachable")` instead of the `return` statement, like https://github.com/golang/go/blob/c5da4fb7ac5cb7434b41fc9a1df3bee66c7f1a4d/src/encoding/xml/xml_test.go#L347-L350. I guess it's the same case here